src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -229,18 +229,19 @@ public Task<CompletionItem> Handle(CompletionItem completionItem, CancellationTo
                 completionItem = completionItem with { Documentation = documentation };
             }
 
+            // We might strip out the commitcharacters for speed, bring them back
+            var container = associatedRazorCompletion.CommitCharacters != null ? new Container<string>(associatedRazorCompletion.CommitCharacters) : null;
```

Comment:
So interesting, the fact the platform even considers commit characters on resolve may be a bug on their side. OR the platform should be respecting the original completionList.CommitCharacters from this completion item. Mind filing a bug on them and then referencing that here?

Reply:
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1505762